doc/tutorials/introduction/openvino_install/openvino_install.markdown

```diff
@@ -0,0 +1,547 @@
+
+Since 2022.2 release OpenVINO does not provide pre-built OpenCV. If you are using OpenVINO with OpenCV functionality there are 2 approaches how to get it:
+
+- Install prebuilt OpenCV from another sources: system repositories, pip, conda, homebrew. It is easy to follow this approach, however it has several limitations:
+    - OpenCV version is out-of-date
+    - OpenCV does not contain G-API module (e.g. some OMZ demos use G-API functionality)
+    - OpenCV does not use available CPU instructions since it has build to cover wide range of hardware
+    - OpenCV does not support Intel TBB, Intel Media SDK
+    - OpenCV DNN module can not use OpenVINO as computational backend
+- Build OpenCV from source code against specific version of OpenVINO. This approach solves the limitations mentioned above.
+
+# Approach #1: Install prebuilt OpenCV {#tutorial_openvino_install_install_prebuilt_opencv}
+
+## Installing on Ubuntu {#tutorial_openvino_install_installing_on_ubuntu}
+
+OpenCV is available from the default Ubuntu repositories:
+
+@code{.bash}
+sudo apt update
+sudo apt install libopencv-dev python3-opencv
+@endcode
+
+## Installing on RHEL {#tutorial_openvino_install_installing_on_rhel}
+
+OpenCV is available from the default RHEL repositories.
+
+1.  Enable CodeReady Linux Builder Repository:
+    @code{.bash}
+    subscription-manager repos --enable codeready-builder-for-rhel-8-x86_64-rpms
+    @endcode
+
+2.  Install OpenCV:
+    @code{.bash}
+    sudo yum update
+    sudo yum install opencv opencv-devel opencv-python
+    @endcode
+
+## Installing on Windows {#tutorial_openvino_install_installing_on_windows}
+
+1.  Download OpenCV windows package on [the official OpenCV releases page](https://github.com/opencv/opencv/releases) Windows archive contains VC postfix, e.g.: `opencv-4.6.0-vc14_vc15.exe`
+2.  Extract the archive.
+3.  Setup environment variables by running the script `opencv\build\setup_vars_opencv4.cmd`
+
+## Installing on macOS {#tutorial_openvino_install_installing_on_macos}
+
+OpenCV could be installed via Homebrew:
+
+@code{.bash}
+brew install opencv
+@endcode
+
+# Approach #2: Build OpenCV against specific version of OpenVINO {#tutorial_openvino_install_build_opencv_against_specific_version_of_openvino}
+
+## Building on Ubuntu {#tutorial_openvino_install_building_on_ubuntu}
+
+### Prerequisites {#tutorial_openvino_install_building_on_ubuntu_prerequisites}
+
+1.  Install OpenVINO according to [the instruction](https://docs.openvino.ai/latest/openvino_docs_install_guides_installing_openvino_linux.html).
+2.  Install the following packages:
+
+    @code{.bash}
+    sudo apt-get install \
+    build-essential \
+    cmake \
+    ninja-build \
+    libgtk-3-dev \
+    libpng-dev \
+    libjpeg-dev \
+    libwebp-dev \
+    libtiff5-dev \
+    libopenexr-dev \
+    libopenblas-dev \
+    libx11-dev \
+    libavutil-dev \
+    libavcodec-dev \
+    libavformat-dev \
+    libswscale-dev \
+    libavresample-dev \
+    libtbb2 \
```

Comment:
OpenVINO uses own TBB.

Also there is WITH_TBB=OFF option below.

Reply:
Fixed.